lib/shared/layers/python-sdk/python/genai_core/documents.py

```diff
@@ -190,9 +208,30 @@ def set_status(workspace_id: str, document_id: str, status: str):
             ":timestampValue": timestamp,
         },
     )
+    update_timestamp(workspace_id, document_id)
+    return response
 
+def update_timestamp(workspace_id: str, document_id: str):
+    timestamp = datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
consider creating a function to return the DDB formatted timestamp since this is used in several places